### PR TITLE
Add test case for ISS incomplete archive import

### PR DESCRIPTION
**Pull request by vsedmik**
This PR just adds negative test case for ISS import of an incomplete export archive, which mocks interrupted data transfer.

### CODE REVIEW

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_satellitesync.py -k incomplete

**Comment by vsedmik** [comment]
Rebased.

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_satellitesync.py -k incomplete

**Review by Griffin-Sullivan — APPROVED** [review]

File: tests/foreman/cli/test_satellitesync.py [hunk 1]
```diff
@@ -1163,6 +1163,91 @@ def test_negative_import_invalid_path(self, module_org, module_target_sat):
...
+        indirect=True,
+    )
+    def test_negative_import_incomplete_archive(
+        self,
+        target_sat,
+        config_export_import_settings,
+        export_import_cleanup_function,
+        function_synced_rh_repo,
+        function_sca_manifest_org,
+        function_import_org_with_manifest,
```
Comment (vijaysawant) [inline tests/foreman/cli/test_satellitesync.py:1179]: Do we need 2 fixtures which returns "org which has manifest uploaded"
fixture reference
`robottelo/pytest_fixtures/component/taxonomy.py` => `function_sca_manifest_org`
`robottelo/tests/foreman/cli/test_satellitesync.py` => `function_import_org_with_manifest`

Will it create 2 different orgs?

File: tests/foreman/cli/test_satellitesync.py [hunk 2]
```diff
@@ -1163,6 +1163,91 @@ def test_negative_import_invalid_path(self, module_org, module_target_sat):
...
+            }
+        )
+        target_sat.cli.ContentView.publish({'id': cv['id']})
+        cv = target_sat.cli.ContentView.info({'id': cv['id']})
+        assert len(cv['versions']) == 1
+        cvv = cv['versions'][0]
+        export = target_sat.cli.ContentExport.completeVersion(
+            {'id': cvv['id'], 'organization-id': function_sca_manifest_org.id}
+        )
+        assert '1.0' in target_sat.validate_pulp_filepath(
```
[the remark this answers is not in the document]
Comment (Griffin-Sullivan) [reply tests/foreman/cli/test_satellitesync.py:1214]: The content view version

File: tests/foreman/cli/test_satellitesync.py (hunk 1 above)
Comment (Griffin-Sullivan) [reply tests/foreman/cli/test_satellitesync.py:1179]: It seems like it creates two different orgs. One is made to export in and the other is what we are importing into. @vsedmik correct me if I'm wrong. I am a ISS noob.

File: tests/foreman/cli/test_satellitesync.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (vsedmik) [reply tests/foreman/cli/test_satellitesync.py:1214]: Correct, for `ContentExport.completeVersion` it is the CV version.

File: tests/foreman/cli/test_satellitesync.py (hunk 1 above)
Comment (vsedmik) [reply tests/foreman/cli/test_satellitesync.py:1179]: Correct again! We need two orgs for that exact reason right now. I have a dream that we have a refactor and move to a completely separate import Satellite, one day.

Comment (vijaysawant) [reply tests/foreman/cli/test_satellitesync.py:1179]: cool, ISS scenario can be perform on same satellite, good to know that.

**Review by vijaysawant — APPROVED** [review]

**Pull request merged**